src/Relude/Extra/Type.hs

```diff
@@ -33,8 +33,8 @@ Note: This must be used with -XTypeApplications
 "Int"
 >>> typeName @String
 "[Char]"
->>> typeName @[Maybe Int]
-"[Maybe Int]"
+>>> typeName @Maybe Int
```

Comment:
Maybe here should be `typeName @(Maybe Int)`? Does it work without parenthesis?

Reply:
indeed